docs_src/microservices/application/V2Migration.md

```diff
@@ -0,0 +1,296 @@
+
+#### SecretStoreExclusive
+
+The `SecretStoreExclusive` section has been removed in EdgeX 2.0. With Edgex 2.0 all SecretStores are exclusive, so the existing `SecretStore` section is all that is required. Services requiring `known secrets` such as `redisdb` must inform the `Security SecretStore Setup` service (via environment variables) that the application service requires the secret added to its SecretStore. See the [Configuring Add-on Service](../../../security/Ch-Configuring-Add-On-Services) section for more details.
+
+#### Clients
+
+The client used for the version validation check has changed to being from Core Metadata, rather than Core Data. This is because Core Data is now optional when persistence isn't required since all Device Services are publishing directly to the EdgeX MessageBus. The configuration for Core Metadata is the only `Clients` entry required, all other (see below) are optional based on use case needs. 
+
+!!! note 
+    The port numbers for all EdgeX services have changed which must be reflected in the `Clients` configuration.
+
+!!! example "Example - Core Metadata client configuration"
+    ```toml
+      [Clients]
+        [Clients.core-metadata]
+        Protocol = 'http'
+        Host = 'localhost'
+        Port = 59881
+    ```
+
+!!! example "Example - All available clients configured with new port numbers"
+    ```toml
+      [Clients]
+        # Used for version check on start-up
+        # Also used for DeviceSevice, DeviceProfile and Device clients
+        [Clients.core-metadata]
+        Protocol = 'http'
+        Host = 'localhost'
+        Port = 59881
+        
+        # Used for Event client which is used by PushToCoreData function
+        [Clients.core-data]
+        Protocol = 'http'
+        Host = 'localhost'
+        Port = 59880
+        
+        # Used for Command client
+        [Clients.core-command]
+        Protocol = 'http'
+        Host = 'localhost'
+        Port = 59882
+        
+        # Used for Notification and Subscription clients
+        [Clients.support-notifications]
+        Protocol = 'http'
+        Host = 'localhost'
+        Port = 59860
+    ```
+
+#### Trigger
+
+The `Trigger` section (previously named `Binding`) has been restructured with `EdgexMessageBus` (previously named `MessageBus`) and `ExternalMqtt` (previously named `MqttBroker` ) moved under it. The `SubscribeTopics` (previously named `SubscribeTopic`) has been moved under the `EdgexMessageBus.SubscribeHost` and `ExternalMqtt` sections. The `PublishTopic` has been moved under the `EdgexMessageBus.PublishHost` and `ExternalMqtt` sections.
+
+##### EdgeX MessageBus
+
+If your Application Service is using the EdgeX MessageBus trigger, you can simply copy the complete `Trigger` configuration from the example below and tweak it as needed. 
+
+!!! example "Example - EdgeX MessageBus trigger configuration"
+
+    ```toml
```

Comment:
should we see the ``` toml part in the display..? This is  something I see throughout the doc for the go and toml snippets.

Reply:
Again, normal markdown doesn't render properly. It renders like this for Github pages:
![image](https://user-images.githubusercontent.com/44779287/125105749-a94ad480-e093-11eb-80f3-766fc194d996.png)